pagopa/io-app
# feat(IT Wallet): [SIW-1035] Add eID to wallet

> [!WARNING]
> Depends on #5965

## Short description
This PR adds a reducer to store the eID (and other credentials) and persist them using `io-react-native-secure-storage`. It also adds the credentials to the wallet screen.

## List of changes proposed in this pull request
- Added `credentials` reducer persisted with `io-react-native-secure-storage`
- eID card added to the wallet screen after issuing
- Credentials added to the wallet screen at startup
- _[Dev mode only]_ Added a section to display the wallet instance state and a button to reset it

## How to test

1. Go through the eID issuance flow and get an eID. Go to the wallet screen and check that the card is present.
2. Restart the app, go the wallet screen and check that the eID card is still present.

https://github.com/user-attachments/assets/bcc38dec-f540-407c-ad9f-89e3c61ce65b
